itou/templates/gps/my_groups.html

```diff
@@ -29,6 +29,10 @@ <h1 class="m-0">Mes bénéficiaires</h1>
                     <i class="ri-user-add-line" aria-hidden="true"></i>
                     <span>Ajouter un bénéficiaire</span>
                 </a>
+            {% else %}
+                <a href="{{ request_new_beneficiary_form_url }}" id="request-new-beneficiary-link" class="btn btn-lg btn-ico btn-primary mt-3 mt-md-0" aria-label="Demander l'ajout d'un bénéficiaire">
```

Comment:
Le `aria-label` n’apporte rien par rapport au label du `<a>`.
```suggestion
                <a href="{{ request_new_beneficiary_form_url }}" id="request-new-beneficiary-link" class="btn btn-lg btn-ico btn-primary mt-3 mt-md-0">
```